<launch>
  <!-- Tester for FDG-->
  <!-- It does not include visualization nodes -->
  <!-- Launch kompai nodes -->
  <!--    -->

  <!-- Launch RFID Node -->
  <node name="rfid_reader" pkg="rfid_node" type="rfid_reader_ros.py"/>

  <!-- We need a map published under /map topic... -->
  
   <!-- Load regions descriptions used by all object trackers -->
  <rosparam command="load" file="$(find rfid_grid_map)/config/FDG3.yaml"/>
  
  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid_deploy.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390200010000000000000000" />
    <arg name="object" value="wallet" />
    
  </include>

  
  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid_deploy.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390200010000000000000001" />
    <arg name="object" value="keys" />
    
  </include>
  
  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid_deploy.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390200010000000000000002" />
    <arg name="object" value="pillbox" />
    
  </include>
  
  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid_deploy.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390200010000000000000003" />
    <arg name="object" value="remote" />
    
  </include>
  
  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid_deploy.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390200010000000000000004" />
    <arg name="object" value="glasses" />
    
  </include>
  
 <!-- Launch ROL Node -->
<node name="rol_srv_node" pkg="rol_server" type="rol_server_node.py" output="screen">  
      <!-- minProb is between 0.0 and 100.0 -->   
      <rosparam param="minProb">0.0</rosparam>        
  </node>

</launch>

